inst/www/flex_dashboard/flexdashboard.js

```diff
@@ -1236,16 +1239,18 @@ var FlexDashboard = (function () {
       if (activeTab.length) {
         activeTab.tab('show');
       } else {
+        // returning to the base page URL without a hash activates first tab
         $('ul.nav a:first').tab('show');
       }
-      FlexDashboardUtils.manageActiveNavbarMenu();
     });
 
     // add a hash to the URL when the user clicks on a tab/page
     $('.navbar-nav a[data-toggle="tab"]').on('click', function(e) {
       var baseUrl = FlexDashboardUtils.urlWithoutHash(window.location.href);
       var hash = FlexDashboardUtils.urlHash($(this).attr('href'));
       var href = baseUrl + hash;
+      // the BS tab plugin handles changing the page and managing the navbar
+      // but flexdashboard needs to update the location and history
```

Comment:
I find this to be a bit redundant given the comment above (but I don't feel strongly about it, so feel free to keep it)

```suggestion
```